mealie/routes/recipe/recipe_crud_routes.py

```diff
@@ -118,20 +119,22 @@ def get_recipe_as_format(self, slug: str, template_name: str, temp_dir=Depends(t
     @router_exports.get("/{slug}/exports/zip")
     def get_recipe_as_zip(self, slug: str, token: str, temp_path=Depends(temporary_zip_path)):
```

Comment:
The issue is here: technically a user can provide the `temp_path` var explicitly which can be used maliciously. You should be able to fix this super easily by just removing the param and calling `temporary_zip_path` in the route:

```python
@router_exports.get("/{slug}/exports/zip")
    def get_recipe_as_zip(self, slug: str, token: str):
        """Get a Recipe and It's Original Image as a Zip File"""
        temp_path = temporary_zip_path()
        ...
```

Reply:
Completely missed that `temp_path` could be passed in by a user.  Should be no problem to fix.